src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -138,6 +138,9 @@ def ensure_bicep_installation(release_tag=None, target_platform=None, stdout=Tru
 
         os.chmod(installation_path, os.stat(installation_path).st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
 
+        _logger.warning("The configuration value of bicep.use_binary_from_path has been set to 'false'.")
```

Comment:
Should we only log warning if the current configuration value is not `false`?

Reply:
I had thought that would never happen after the first time but forgot about the `az bicep upgrade` command. Thanks for catching that!

Done 👍